server.js

```diff
@@ -18,10 +18,62 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Think happy thoughts, think happy thoughts");
 });
+//////////////////
+const ThoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: (() => new Date())()
+  }
+})
+
+const Thought = mongoose.model("Thought", ThoughtSchema);
+
+// Get 20 of the latest thoughts added
+app.get("/thoughts", async (req, res) => {
+  try {
+    const thoughts = await Thought.find({}).sort({ createdAt: -1 }).limit(20).exec()
```

Comment:
Unfortunately my thoughts are not posted in descending order (however, your posts are at the bottom of the page, really strange!). Maybe it has to do with the time stamp?